var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -47,6 +47,8 @@ class PyPybind11(CMakePackage):
     version('2.1.0', '3cf07043d677d200720c928569635e12')
 
     depends_on('py-pytest', type='test')
+    depends_on('py-setuptools', type='build')
```

Comment:
why did you need to add this?

Reply:
The setup.py needs the py-setuptools to function properly